src/librustdoc/html/render.rs

```diff
@@ -2899,10 +2945,12 @@ fn render_impl(w: &mut fmt::Formatter, cx: &Context, i: &Impl, link: AssocItemLi
 
 fn item_typedef(w: &mut fmt::Formatter, cx: &Context, it: &clean::Item,
                 t: &clean::Typedef) -> fmt::Result {
+    let indent = format!("type {}{:#} ", it.name.as_ref().unwrap(), t.generics);
+    let indent = repeat(" ").take(indent.len()).collect::<String>();
```

Comment:
Oh yeah, I remember you talking about that on IRC. This is a pattern used more broadly than my changes, so I'd be more comfortable taking that to a separate PR? I dunno what the general consensus on that would be, though.